src/code.h

```diff
@@ -46,9 +46,14 @@ struct CodeState {
     // the offset in the current body. It is only valid while coding
     Stat OffsBody;
 
+    // a linked list of further OffsBody values, used when coding
+    // nested function expressions
+    Obj OffsBodyList;
```

Comment:
I think this needs InitGlobalBag adding to code.c:InitKernel?

Reply:
Ah, good one... For a moment I was sweating 😅 However, all instances of `struct CodeState` are allocated on the stack, so I think `InitGlobalBag` is neither possible nor necessary. Or am I missing something?

Reply:
I'll be honest, my reasoning was CodeResult, which is stored just below, is `InitGlobalBag`ed on code.c:3237

Reply:
That InitGlobalBag call gets removed by this PR